src/resources/postcss/tribe-common-admin/_fields.pcss

```diff
@@ -18,4 +18,23 @@
 			width: auto;
 		}
 	}
-}
\ No newline at end of file
+
+	&.tribe-field-color {
```

Comment:
Any particular reason why we're using multiple selectors? If we go with a unique class there wouldn't be enough?

Reply:
All fields are wrapped within the `.tribe-field` class, but since `.wp-color-result.button` and `.wp-color-result-text` are global (WordPress) classes, and could be used in other types of fields in the future, I felt it was best to narrow it down to just the `.tribe-field-color` field so it doesn't affect the styling elsewhere. The default style of the color picker is noticeably different.

Reply:
Gotcha, thanks for the clarification 👍